File: model/paciente.py
from utils.db import db
from dataclasses import dataclass

@dataclass
class Paciente(db.Model):
    __tablename__ = 'paciente'
    id_paciente = db.Column(db.Integer, primary_key=True, autoincrement=True)
    id_persona = db.Column(db.Integer, db.ForeignKey('persona.id_persona'))
    codigo_paciente = db.Column(db.String(100))
    antecedentes = db.Column(db.String(255))

    persona = db.relationship('Persona', backref='paciente')

    def __init__(self, id_persona, codigo_paciente, antecedentes):
        self.id_persona = id_persona
        self.codigo_paciente = codigo_paciente
        self.antecedentes = antecedentes

File: v2-microservicios/sisvita-app/auth-server/src/main/java/sm/dsw/sisvita/auth/model/Paciente.java
package sm.dsw.sisvita.auth.model;

import jakarta.persistence.*;
import lombok.*;
import java.util.Date;

@Entity
@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "paciente")
public class Paciente {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id_paciente;

    @OneToOne
    @JoinColumn(name = "id_persona", referencedColumnName = "id_persona")
    private Persona persona;

    @Column(name = "codigo_paciente", unique = true)
    private String codigo_paciente;

    @Column(name = "antecedentes")
    private String antecedentes;
}
